ipykernel/eventloops.py

```diff
@@ -125,7 +125,6 @@ def loop_qt4(kernel):
 @register_integration('qt', 'qt5')
 def loop_qt5(kernel):
     """Start a kernel with PyQt5 event loop integration."""
-    os.environ['QT_API'] = 'pyqt5'
```

Comment:
If this is not set, then there's no way to select the right Python bindings to create the event loop (unless users set `QT_API` by hand).

Therefore, I'm -1 on this one.

Reply:
OK, changed it to set if unset. This also prevents it from being overwritten in every loop (as it was originally).